unumpy/tests/test_numpy.py

```diff
@@ -281,3 +281,41 @@ def test_array_creation(backend, method, args, kwargs):
         assert ret.dtype == ndt(dtype)
     else:
         assert ret.dtype == dtype
+
+
+@pytest.mark.parametrize(
+    "method, args, kwargs",
+    [
+        # (np.fft.fft, (4 * np.eye(4),), {}),
+        (np.fft.ifft, ([[1, 2], [3, 4]],), {}),
+        (np.fft.fft2, (4 * np.eye(4),), {}),
+        (np.fft.ifft2, (4 * np.eye(4),), {}),
+        (np.fft.fftn, (4 * np.eye(4),), {}),
+        (np.fft.ifftn, (4 * np.eye(4),), {}),
+        (np.fft.rfft, (4 * np.eye(4),), {}),
+        (np.fft.irfft, (4 * np.eye(4),), {}),
+        (np.fft.rfft2, (4 * np.eye(4),), {}),
+        (np.fft.irfft2, (4 * np.eye(4),), {}),
+        (np.fft.rfftn, (4 * np.eye(4),), {}),
+        (np.fft.irfftn, (4 * np.eye(4)), {}),
+        (np.fft.hfft, (4 * np.eye(4)), {}),
+        (np.fft.ihfft, (4 * np.eye(4)), {}),
+        (np.fft.rfftn, (4 * np.eye(4)), {}),
+        (np.fft.fftfreq, (10), {}),
+        (np.fft.rfftfreq, (10), {}),
```

Comment:
These are missing the commas that makes `args` a tuple.
```suggestion
        (np.fft.fftfreq, (10,), {}),
        (np.fft.rfftfreq, (10,), {}),
```

I think that code coverage may be crashing because the ast is invalid?

Reply:
Actually, a bunch of the lists above are missing the comma as well.

Reply:
Well, the AST is valid, just that `*a` isn't valid when `a` isn't a sequence or iterable.